packages/store/src/StoreCore.sol

```diff
@@ -95,8 +95,8 @@ library StoreCore {
    */
   function registerSchema(uint256 tableId, Schema valueSchema, Schema keySchema) internal {
     // Verify the schema is valid
-    valueSchema.validate();
-    keySchema.validate();
+    valueSchema.validate(false);
+    keySchema.validate(true);
```

Comment:
I'm not sure if the following compiles, but it would be more readable in this case. Should test if it compiles before committing though
```suggestion
    valueSchema.validate({ allowEmpty: false });
    keySchema.validate({ allowEmpty: true });
```

Reply:
haha you merged it right  before I pushed the changes. It does compile

Reply:
(wanted to do as a manual commit since it's used in 2 other places)